tutorials/telemetry_hello_world.kp/orig_src/Telemetry Hello World.ipynb

```diff
@@ -279,133 +268,149 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's extract a histogram from the submissions:"
+    "Let's extract a histogram of GC_MARK_MS (time spent running JS garbage collection mark phase) from the submissions:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "scrolled": true
    },
    "outputs": [],
    "source": [
-    "histograms = get_pings_properties(pings, \"payload/histograms/GC_MARK_MS\", with_processes=True)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "The API returns three distinct histograms for each submission:\n",
-    "- a histogram for the parent process (*GC_MARK_MS_parent*)\n",
-    "- an aggregated histogram for the child processes (*GC_MARK_MS_children*)\n",
-    "- the aggregate of the parent and child histograms (*GC_MARK*)"
+    "histograms = pings_dataset \\\n",
+    "    .select(GC_MARK_MS_content='payload.processes.content.histograms.GC_MARK_MS.values',\n",
+    "            GC_MARK_MS_parent='payload.histograms.GC_MARK_MS.values') \\\n",
+    "    .records(sc, sample=0.05)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "- `payload.histograms.GC_MARK_MS.values` is a path to the GC_MARK_MS values of the parent (main) process\n",
+    "- `payload.processes.content.histograms.GC_MARK_MS.values` is a path to the GC_MARK_MS values of the child processes\n",
+    "\n",
     "Let's aggregate the histogram over all submissions and plot it:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "scrolled": false
    },
    "outputs": [],
    "source": [
-    "def aggregate_arrays(xs, ys):\n",
-    "    if xs is None:\n",
-    "        return ys\n",
-    "    \n",
-    "    if ys is None:\n",
-    "        return xs\n",
-    "    \n",
-    "    return xs + ys\n",
-    "    \n",
-    "aggregate = histograms.map(lambda p: p[\"payload/histograms/GC_MARK_MS\"]).reduce(aggregate_arrays)\n",
-    "aggregate.plot(kind=\"bar\", figsize=(15, 7))"
+    "def aggregate_series(s1, s2):\n",
```

Comment:
Interesting use of series. I've always imagined series to be ordered in time, which would make histograms a [cross-sectional view](https://en.wikipedia.org/wiki/Cross-sectional_study) of the data. After reading through the [pandas.Series](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.html) api, the data-structure makes sense since we're interested in sorted bucket labels and a series is generally an indexed, array-like data.